.github/workflows/androidKtlint.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         if: steps.cache-ktlint.outputs.cache-hit != 'true' && steps.android-hash.outputs.hash != 'no-android-files'
         working-directory: ./ReadersAndroidJetpack
         run: |
-          ./gradlew ktlintCheck
+          ./gradlew ktlintCheck --stacktrace
```

Comment:
Is this what triggered 400+ lines of exception backtrace reporting in the run output? It made finding the actual lint issues in the output a bit challenging.